docs/_includes/links.md

```diff
@@ -6,8 +6,8 @@
 
 <!-- Classes and interfaces -->
 
-[FluidContainer]: {{< relref "fluidcontainer.md" >}}
-[IFluidContainer]: {{< relref "ifluidcontainer.md" >}}
+[FluidContainer]: {{< relref "/docs/apis/fluid-framework/fluidcontainer.md" >}}
+[IFluidContainer]: {{< relref "/docs/apis/fluid-framework/ifluidcontainer.md" >}}
```

Comment:
Is there some reason the full path is needed now?

Reply:
For standard in-doc links, `apiref` should continue to work, but for these I think we will have to be more explicit.